client/src/Movies/Movie.js

```diff
@@ -11,7 +12,7 @@ export default class Movie extends Component {
 
   componentDidMount() {
     // change this line to grab the id passed on the URL
-    const id = 1;
+    const id = this.props.match.params.id;
```

Comment:
I want you to Review this a bit more. Since this is a bit complicated, but at the same time, it's a very important line of code